commands/src/main/java/net/apartium/cocoabeans/commands/CommandProcessingContext.java

```diff
@@ -51,4 +53,6 @@ public interface CommandProcessingContext {
      */
     RequirementResult senderMeetsRequirement(Requirement requirement);
 
+    void report(Object source, @NotNull BadCommandResponse response);
```

Comment:
/**
 * Report a problem with command parsing, such as invalid arguments
 * @argument source reporter of the problem
 * @argument response error description object
*/